.eslintrc

```diff
@@ -7,5 +7,8 @@
         "ecmaVersion": 8,
         "sourceType": "module",
         "ecmaFeatures": {}
+    },
+    "rules": {
+      "no-throw-literal": 0
```

Comment:
Can you use "off" here instead of '0'? Having a 0/1/2 is non-descript, where strings make it clear what youre trying to do without having to google ESLint man pages